src/sagas/formSaga.js

```diff
@@ -0,0 +1,27 @@
+import { call, put, takeLatest } from 'redux-saga/effects';
+import candidateInfoPostApi from 'apis/candidateFormApi';
+import { CANDIDATE_FORM_ACTIONS } from 'constants/actionConstants';
+import {
+  candidateFormSuccessAction,
+  candidateFormFailureAction,
+} from 'actions/candidateFormActions';
+
+export function* candidateFormSaga(action) {
+  try {
+    const response = yield call(candidateInfoPostApi, action.payload);
+
+    if (response.status >= 200 && response.status < 300) {
+      const { data } = response.data;
+      yield put(candidateFormSuccessAction(data));
+    } else {
+      throw response;
+    }
+  } catch (error) {
+    yield put(candidateFormFailureAction(error));
+  }
+}
+
+// watcher saga
+export default function* formSaga() {
```

Comment:
Codacy has a fix for the issue: [Missing space before *.](https://app.codacy.com/gh/joshsoftware/OnCoT-Frontend/pullRequest?prid=7073195)
```suggestion
export default function * formSaga() {
```